Resources/Locale/en-US/_Nyano/store/uplink-catalog.ftl

```diff
@@ -0,0 +1,8 @@
+uplink-kanabou-name = Kanabou
+uplink-kanabou-desc = A weapon for those who wish to forgo subtlety. Particularly suited to oni.
```

Comment:
Extra lines?

Reply:
well technically is adding them back since they where deleted and the uplink list doesn't have the .ftl for the kanabou and Rickenbacker.